### PR TITLE
feat: add avatar to user model

### PR DESCRIPTION
**Pull request by taylrj**
Address [TWREPORTER-119](https://twreporter-org.atlassian.net/browse/TWREPORTER-119).

 https://github.com/twreporter/keystone/pull/202 is required by this change.

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -1,15 +1,27 @@
 var keystone = require('@twreporter/keystone');
 var transform = require('model-transform');
 var Types = keystone.Field.Types;
+var config = require('../config');
 
 var User = new keystone.List('User');
 
+const bucket = config['options']['gcs config']['bucket']['resizedImage'];
```
Comment (babygoat) [inline models/User.js:8]: I supposed we should use different bucket?

Comment (taylrj) [reply models/User.js:8]: Yes, I second that.

How about creating a new bucket 'twreporter-avatars' for production usage, what do you say ?

And this line will be:
`bucket = config['options']['gcs config']['bucket']['avatar']`

File: models/User.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ User.add({
     initial: true,
     bucket: bucket,
     destination: 'avatars',
-    publicRead: true
+    publicRead: true,
+    autoCleanup: true,
```
Comment (babygoat) [inline models/User.js:22]: If the name is the same, do we really need to delete and write the same data?

Comment (taylrj) [reply models/User.js]: No, this commit has been removed.

Updated.

File: models/User.js (hunk 1 above)
Comment (babygoat) [reply models/User.js:8]: I suppose we could just put in `twreporter-multimedia`?

Comment (taylrj) [reply models/User.js]: Updated in https://github.com/twreporter/plate/pull/263/commits/0366f0d53fcf3fec0228e985313b7e2ef5a15595 and https://github.com/twreporter/kubernetes-config/pull/40.

**Comment by taylrj** [comment]
Update @twreporter/keystone with changes in https://github.com/twreporter/keystone/pull/202

**Comment by babygoat** [comment]
LGTM

**Pull request merged**